### PR TITLE
Update VCardConverter.php

### PR DESCRIPTION
**Pull request by sash04ek**
VCARD 3.0 has 'X-ADDRESSBOOKSERVER-MEMBER' property and VCARD 4.0 has 'MEMBER' property.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
Please add some description. Thx

**Comment by sash04ek** [comment]
VCARD 3.0 has 'X-ADDRESSBOOKSERVER-MEMBER' property and VCARD 4.0 has 'MEMBER' property.

File: lib/VCardConverter.php [hunk 1]
```diff
@@ -141,7 +141,10 @@ protected function convertProperty(Component\VCard $input, Component\VCard $outp
                         $newProperty = $output->createProperty('X-ADDRESSBOOKSERVER-KIND', 'GROUP');
                         break;
                 }
+            } elseif ('MEMBER' === $property->name) {
```
Comment (phil-davis) [inline lib/VCardConverter.php:144]: https://en.wikipedia.org/wiki/VCard#Properties says that MEMBER is a property of vCard V4, But is "Undefined" in V3 and V2. 

Please provide a reference for this being a property of V3.

Comment (sash04ek) [reply lib/VCardConverter.php:144]: for vCard V2 and V3 - X-ADDRESSBOOKSERVER-MEMBER property is used, but for vCard V4 - MEMBER property is used

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@sash04ek thanks for the contribution

**Pull request merged**